modules/lead/dashboard/dashboard-values.tpl

```diff
@@ -1,7 +1,7 @@
 elasticsearch:
 %{ if local == "false" }
   volumeClaimTemplate:
-    storageClassName: gp2
+    storageClassName: ${k8s_storage_class}
```

Comment:
If you set this in environment-local you can move this out of the `if local` condition

Reply:
the `else` condition also specifies a `volumeClaimTemplate` with some other configuration.  I think this will look messier if I move this out, since I'll need another `if` condition.  I can try it if you'd like.